vmaas-go/base/utils/config.go

```diff
@@ -88,14 +94,31 @@ func initDB() {
 	Cfg.DBPassword = clowder.LoadedConfig.Database.Password
 }
 
-func initAPI() {
+func initEnvDB() {
+	Cfg.DBHost = os.Getenv("POSTGRESQL_HOST")
+	Cfg.DBName = os.Getenv("POSTGRESQL_DB")
+	Cfg.DBPort, _ = strconv.Atoi(Getenv("POSTGRESQL_PORT", "5432"))
+	Cfg.DBSslMode = Getenv("POSTGRESQL_SSL_MODE", "prefer")
+	Cfg.DBSslRootCert = os.Getenv("POSTGRESQL_SSL_ROOT_CERT_PATH")
+	Cfg.DBUser = os.Getenv("POSTGRESQL_USER")
+	Cfg.DBPassword = os.Getenv("POSTGRESQL_PASSWORD")
+}
+
+func initClowderAPI() {
 	Cfg.PublicPort = *clowder.LoadedConfig.PublicPort
 	Cfg.PrivatePort = *clowder.LoadedConfig.PrivatePort
 	Cfg.MetricsPort = clowder.LoadedConfig.MetricsPort
 	Cfg.MetricsPath = clowder.LoadedConfig.MetricsPath
 }
 
-func initEndpoints() {
+func initEnvAPI() {
+	Cfg.PublicPort = 8000
+	Cfg.PrivatePort = 10000
+	Cfg.MetricsPort, _ = strconv.Atoi(Getenv("PROMETHEUS_PORT", "-1"))
```

Comment:
Why don't set the default to 9000?

Reply:
The Go version has the ability to turn it off, so I've set it as a default. Do you want to have it enabled by default?

Reply:
Set to 9000 by default